### PR TITLE
LPS-146959 | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-146959
 
 cc @david-gutierrez-mesa 
 cc @Tim-Cao 
 
 Hi Tim, I've added all testcases at the same commit because I'm having an issue on my IDE. The next one will have each test on its commit.

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenus.testcase [hunk 1]
```diff
@@ -1267,7 +1340,7 @@ definition {
 		}
 	}
 
-}	@description = "This is a test for LPS-143841. View warning is shown after remove a Display Page Template for an item in Navigation Menu."
+	@description = "This is a test for LPS-143841. View warning is shown after remove a Display Page Template for an item in Navigation Menu."
 	@priority = "4"
 	test ViewWarningForItemAfterRemoveDisplayPageTemplate {
```
Comment (ozysouza) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenus.testcase:1345]: This whole test should be on the pull request. I'm searching to solve it.

**Comment by Tim-Cao** [comment]
Just start reviewing

**Comment by Tim-Cao** [comment]
Hi @ozysouza, see changes https://github.com/liferay-echo/liferay-portal/compare/134e7d7331c284a499c2e0accbe35ae5d39d5871...f6b947ca672d96e0767062a8b4bdd1a574d26f26

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**